samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/TotalScoreUpdateListener.java

```diff
@@ -256,25 +256,8 @@ public boolean saveTotalScores(TotalScoresBean bean)
       }
 
       if (hasNumberFormatException) {
-    	  if (bean.getPublishedAssessment() != null 
```

Comment:
This is already done earlier in the method and repeated here. Since bean hasn't changed it can be removed.